target/linux/ramips/base-files/etc/board.d/02_network

```diff
@@ -31,6 +31,10 @@ ramips_setup_interfaces()
 	local board="$1"
 
 	case $board in
+	adslr,g7)
+		ucidef_add_switch "switch0" \
+			"0:lan" "1:lan" "2:lan" "3:lan" "4:wan" "6@eth0"
+		;;
```

Comment:
As indicated in the initial PR, you can merge this with the subsequent block by just adding `adslr,g7|\` there.

Reply:
I think this must be done. I found that using adslr, g7|\ during the test, will cause the compiled firmware to fail to get the IP address and cannot be set.

Reply:
Well, at least here in your description you added a space between "adslr" and "g7", which would have made it fail.
If you implement the string correctly, it will work. Just put in in front of afoundry,ew1200 .

Reply:
I will do this according to what you said, but I can't get the IP address. I didn't find the space you said with gedit or vi. This question is very strange.

Reply:
The space was only in your comment here on GitHub, so don't bother with that.
Just put `adslr,g7|\` in an empty line before `afoundry,ew1200|\` and remove the block above.